### PR TITLE
[Issuee#398] Edit JavaScript: DataTypes

### PR DESCRIPTION
**Pull request by gracenn1998**
### Description

Issue #398 
Edit the existing entry for JavaScript: Data Types:
- Replace usage of 1st- and 2nd-person pronouns with neutral, manual-like language.
- Code block/ Code byte examples

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own (for my updated parts).
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@gracenn1998, I'll be doing the first review of this.

File: content/javascript/concepts/data-types/data-types.md
Comment (SSwiniarski) [inline content/javascript/concepts/data-types/data-types.md:3]:
```suggestion
Description: 'Data types are how data in programming is classified. In JavaScript, there are eight fundamental data types: Number, BigInt, String, Boolean, Null, Undefined, Symbol, and Object.'
```

File: content/javascript/concepts/data-types/data-types.md
Comment (SSwiniarski) [inline content/javascript/concepts/data-types/data-types.md:17]:
```suggestion
Data types are how data in programming is classified. In JavaScript, there are eight fundamental data types:
```

**Comment by gracenn1998** [comment]
@SSwiniarski Thank you 🙇‍♀️ 
I have not expected to get a reviewer that quickly! So I added a force-commit to fix some enters, sorry if it cause any inconveniences

File: content/javascript/concepts/data-types/data-types.md
Comment (SSwiniarski) [inline content/javascript/concepts/data-types/data-types.md:43]: This might be better as a comment in the below code block. All the other descriptive text is in code comments.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@gracenn1998, I had a couple of comments. Resolve them and I'll ask @sonnynomnom if we need a second review before merging.

**Review by SSwiniarski — APPROVED** [review]
@gracenn1998, everything seems to be good for review number two. @Dusch4593, @yangc95?

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:47]:
```suggestion
BigInt is necessary for big numbers because they are unreliable with Number type:

```codebyte/js
console.log(9999999999999999);
console.log(9999999999999999n);
```
```

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:32]:
```suggestion
At first, eight types may not seem like that many, but they are the foundation that helps programmers build a whole new world.
```

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:52]:
```suggestion
// Strings in JavaScript can be defined with either '' or ""
```

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:54]:
```suggestion
var message = "You are doing great! Keep studying!";
```

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:60]:
```suggestion
// Boolean is for truthy or falsy values
```

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:77]:
```suggestion
// Objects can have fields with different values and types inside
```

File: content/javascript/concepts/data-types/data-types.md
Comment (yangc95) [inline content/javascript/concepts/data-types/data-types.md:72]:
```suggestion

Null and Undefined are both for the absence of a value, but they have different meanings.

```js
// Undefined means there should be some values, but it is undefined now
var finishCourseTime = undefined;

// Null means there is no value here
var finishStudyingDate = null;
```
```

**Review by yangc95 — CHANGES_REQUESTED** [review]
Second review underway! Please tag me once you've looked over my suggestions and comments.

Thanks for your contribution! 💡

**Comment by gracenn1998** [comment]
@yangc95 I have updated the suggested changes~

**Review by yangc95 — APPROVED** [review]

**Comment by sonnynomnom** [comment]
@gracenn1998 Make sure to add a GitHub profile picture so you will be listed as one of the contributors!

**Pull request merged**
